pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "ecological"
+version = "2.0.0"
+description = "Map a Python configuration from environment variables"
+authors = [
+    "João Santos <jmcs@jsantos.eu>",
+    "Marcin Zaremba <mrcn.zrmb@gmail.com>"
+]
+license = "MIT"
+readme = "README.rst"
+homepage = "https://github.com/jmcs/ecological"
+repository = "https://github.com/jmcs/ecological"
+classifiers = [
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent"
+]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+dataclasses = "^0.6.0"
+
+[tool.poetry.dev-dependencies]
+tox = "^3.12"
+pytest = "^4.6"
+pytest-cov = "^2.7"
+black = {version = "^18.3-alpha.0",allows-prereleases = true}
+rope = "^0.14.0"
+flake8 = "^3.7"
+codacy-coverage = "^1.3"
```

Comment:
As it is a dependency for Travis only, I would recommend adding it in `.travis.yml` rather than here. It may also make dependency resolution faster for Poetry. :wink: